tests/storage/hotplug.go

```diff
@@ -833,10 +851,15 @@ var _ = SIGDescribe("Hotplug", func() {
 			})
 		})
 
-		Context("VMI only", func() {
+		Context("[Serial]VMI migration", func() {
```

Comment:
does this need to be serial?

Reply:
The test relies on the workaround for the issue with CPU manager and runc: there is a need to ensure that VMI will be migrated to the node **without** cpu manager. For that I do the manipulations with the nodes and set them schedulable/unschedulable as needed. If I understand correctly, the `Serial` label here will prevent other tests to run in parallel with this one. Potentially such nodes manipulations may affect other tests I guess. Or do you think it is not needed in this case?

Reply:
ohh interesting. i think i missed that the node stuff was going on.

There's another way to do this that's super simple and probably doesn't require setting nodes as unscheduable.

- Pick a single node out that you want to run the vmi on and add a unique label to the node
- set a node selector on the VMI to ensure it get's scheduled to that node
- once the VMI is online, add that label to the node you want the VMI to migrate to
- the migration's target pod will then choose that node to run on